llama_index/data_structs/struct_type.py

```diff
@@ -83,6 +83,8 @@ class IndexStructType(str, Enum):
     SQL = "sql"
     # for KG index
     KG = "kg"
+    SIMPLE_KG = "simple_kg"
+    NEBULAGRAPH = "nebulagraph"
```

Comment:
nit: where are these used?

Reply:
I think this just follows the same structure for the vector store registry -- is that registry used anywhere?